voting-hir/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+use jormungandr_lib::{crypto::account::Identifier, interfaces::Value};
+
+pub type VotingGroup = String;
+
+/// Define High Level Intermediate Representation (HIR) for voting
+/// entities in the Catalyst ecosystem.
+///
+/// This is intended as a high level description of the setup, which is not
+/// enough on its own to spin a blockchain, but it's slimmer, easier to understand
+/// and free from implementation constraints.
+///
+/// You can roughly read this as
+/// "voting_key will participate in this voting round with role voting_group and will have voting_power influence"
+pub struct VotingHIR {
+    pub voting_key: Identifier,
```

Comment:
Is `voting_key` a user Identifier ? If so maybe it is better to rename it to `users_voting_key` to reflect that it belongs to user.
Otherwise I think we should provide a description does it mean.

Reply:
I mean, who should this voting key belong to otherwise? If we rename the struct to `VoterHIR` would this be clearer?

Reply:
Yes, it looks better to me